docs/advanced-configuration.md

```diff
@@ -145,3 +195,7 @@ autodetect:
   istio: true
   prometheus: true
 ```
+
+## Override underlying OpenTelemetry agent configuration
+
+If you want to use your own OpenTelemetry Agent configuration, you can override it by providing a custom configuration in `agent.config` parameter in the values.yaml which will be merged into the default agent configuration, list parts of the configuration (e.g. `service.pipelines.logs.processors`) to be fully re-defined.
```

Comment:
If you want to use your own OpenTelemetry Agent configuration, you can override it by providing a custom configuration in the `agent.config` parameter in the values.yaml, which will be merged into the default agent configuration, list parts of the configuration (for example, `service.pipelines.logs.processors`) to be fully re-defined.